### PR TITLE
OCPQE-15783: Add nginx container as a service

### PR DESCRIPTION
**Pull request by mhanss**

### CODE REVIEW

File: images/fcos-bastion-image/root/usr/share/bastion_services/nginx/nginx_systemd.conf [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# systemd nginx conf
+PODMAN_NGINX_IMAGE=quay.io/openshifttest/nginx-alpine:armbm
+PODMAN_NGINX_BASEDIR=/var/opt/nginx
```
Comment (aleskandro) [inline images/fcos-bastion-image/root/usr/share/bastion_services/nginx/nginx_systemd.conf:3]:
```suggestion
PODMAN_NGINX_BASEDIR=/var/opt/html
```

To stay in sync with the other code that will add content there.

**Review by aleskandro — COMMENTED** [review]
Hi @mhanss thanks.

Just a nit to keep the code base in sync with the other services pushing content to be served by nginx.

Other than that, lgtm

File: images/fcos-bastion-image/root/usr/share/bastion_services/nginx/nginx_systemd.conf (hunk 1 above)
Comment (mhanss) [reply images/fcos-bastion-image/root/usr/share/bastion_services/nginx/nginx_systemd.conf]: Done

File: images/fcos-bastion-image/root/usr/lib/systemd/system/nginx.service [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+RequiresMountsFor=%t/containers
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=/usr/share/bastion_services/nginx/nginx_systemd.conf
+Restart=on-failure
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/%n.ctr-id
+ExecStart=/usr/bin/podman run -d --name nginx-container -p 80:80/tcp -p 443:443/tcp \
+    -v ${PODMAN_NGINX_BASEDIR}/html:/usr/share/nginx/html \
```
Comment (mhanss) [inline images/fcos-bastion-image/root/usr/lib/systemd/system/nginx.service:14]: @aleskandro here I have already added html folder, what should be the desire folder structure `/var/opt/html/html` or `/var/opt/html/`?

Comment (mhanss) [reply images/fcos-bastion-image/root/usr/lib/systemd/system/nginx.service]: I have made the changes as `/var/opt/html/`.

**Comment by aleskandro** [comment]
Hi @mhanss this lgtm. Do you have any test of the unit running on an fcos installation rebased to the image built by this PR?

**Comment by mhanss** [comment]
> Hi @mhanss this lgtm. Do you have any test of the unit running on an fcos installation rebased to the image built by this PR?

I have tested it locally and it was working fine.

**Pull request merged**